---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Being trained in a molecular biology lab during my PhD has established my solid knowledge and skills in experimental cancer biology, which is evident by my two first author publications in *PNAS* and *JBC*. With the advancing of next-generation sequencing technique, the ability to analyze the vast amount of data is demanding. At the fourth year of my PhD (2012), I developed a strong interest in analyzing next-generation sequencing data. I started to teach myself linux, python and R programming languages.  I put my learning notes on my [blog](http://crazyhottommy.blogspot.com/). Now it attracts more than 6,000 views per month. The learning process was painful but rewarding. I finished my third project on studying distal regulatory enhancers by analyzing various high-throughput sequencing data including ChIP-seq, RNA-seq, GRO-seq and ChIA-PET. I learned quite a bit on epigenomic data analysis with self -bioinformatics training during my PhD.   

After joining MD Anderson Cancer Center for my post-doctoral training in [Dr.Roel Verhaak’s](https://www.jax.org/research-and-faculty/faculty/roel-verhaak) lab, I extended my bioinformatics skills in integrating analysis of TB size sequencing data sets. Verhaak lab is well known for studying genomic alterations of brain tumor by analyzing large panels of RNA-seq and DNA-seq data. I gained extensive experience in handling large-scale genomic data and pipelining workflows. I also gained intimate familiarities with public data sets such as TCGA. I once processed over 1000 TCGA whole-genome sequencing samples (~300G in size each sample) to identify structural variations. I have put my analysis notes on [github](https://github.com/crazyhottommy). Roel moved to JAX lab for genomic medicine in Oct 2016, I could not move because of family issues. My little girl Phoebe was born in August 2016! She is very cute and joyful:)

To apply my skills to translational lung cancer research, I joined [Dr. Andrew Futreal](https://gsbs.uth.edu/faculty/faculty-directory/faculty-profiles.htm?id=bc44b01e-38b8-4e9f-ab8b-627c05c4b708) and [Dr.Jianjun Zhang’s](http://faculty.mdanderson.org/Jianjun_Zhang/) lab shortly to study tumor heterogeneity in lung cancer by analyzing multi-region whole-exome sequencing data and DNA methylation array data. At the same time, I am jointly supervised by [Dr.Kunal Rai](http://railab.org/people.html) in analyzing a large scale in-house generated ChIP-seq data set from melanoma. Now, with both experimental and computational biology skills on my belt, I am ready to tackle cancer biology problems with my expertise. My ultimate goal is to make new discoveries in cancer biology and translate the knowledge into effective new therapeutic strategies to benefit cancer patients in the end.
I also have a great interest in promoting open science and reforming bioinformatics education. I frequently share my thoughts on twitter @tangming2005 and tips in my blog post. I am a certified instructor for [software carpentry](https://software-carpentry.org/) and [data carpentry](http://www.datacarpentry.org/). I once served as an instructor for the software Carpentry workshop held in the University of Miami to teach biologists Unix and R for data analysis. I have put other training experiences on my [linkedin page](https://www.linkedin.com/in/ming-tang-40650014).

